.github/dependabot.yml

```diff
@@ -0,0 +1,60 @@
+version: 2
```

Comment:
Consider spelling out the strategy/intention based on your commit comment in the YAML. This makes it easier to see for others.